.yarn/patches/jest-message-util-npm-29.7.0-7f88b6e8d1.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/build/index.js b/build/index.js
+index d1cff260e9c9c17b57efde19268d563c35542682..a5c8543f3aa21f5a031f204e9f4c4a6d264cdabc 100644
+--- a/build/index.js
++++ b/build/index.js
+@@ -339,9 +339,13 @@ const getTopFrame = lines => {
+     const parsedFrame = stackUtils.parseLine(line.trim());
+     if (parsedFrame && parsedFrame.file) {
+       if (parsedFrame.file.startsWith('file://')) {
+-        parsedFrame.file = (0, _slash.default)(
+-          (0, _url.fileURLToPath)(parsedFrame.file)
+-        );
++        try {
```

Comment:
unrelated?

Reply:
Jest was hiding some errors. This makes it allow to see them.